templates/no_fill_last.txt

```diff
@@ -1,6 +1,6 @@
 Hello ###LOGIN###,
 
-The last task you recorded in PHPReport was on ###LAST_TASK_DATE###.
+Last week you didn't record any task in the days ###LIST_OF_DATES###.
 
 In three days, you will have gone one month without reporting hours
 in PHPReport. This is much more time than can be handled.
```

Comment:
The text in the emails doesn't make sense with the new feature (e.g. this was supposed to be sent when the person hadn't reported tasks in 27 days). Same for the other txts.

Reply:
I've simplified the templates a bit to make more sense. In our particular case, the real emails will be configured in the host as they contain specific details that only make sense to Igalia.